app/elements/common/coduno-iron-ajax.html

```diff
@@ -3,15 +3,14 @@
 <dom-module id="coduno-iron-ajax">
 	<template>
 		<iron-ajax
-		  id="ajaxRequest"
-		  method="{{method}}"
-		  content-type="{{contentType}}"
-		  handle-as="{{handleAs}}"
-		  last-response="{{lastResponse}}"
-		  headers="{{headers}}"
-		  url="{{url}}"
+			id="ajaxRequest"
```

Comment:
replace `{{}}` with `[[]]` except for `last-response`. Add `auto="[[auto]]"`. Add a provider with the `baseUrl`(like in providers) and `url="[[baseUrl]]/[[path]]"`